processor/cumulativetodeltaprocessor/factory.go

```diff
@@ -42,11 +42,12 @@ func NewFactory() component.ProcessorFactory {
 func createDefaultConfig() config.Processor {
 	return &Config{
 		ProcessorSettings: config.NewProcessorSettings(config.NewID(typeStr)),
+		MonotonicOnly:     true,
```

Comment:
Do we want to add a line in the README to the default value fo r`MonotoincOnly` is true?